app/javascript/src/NewService/components/ServiceManualForm.jsx

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import React from 'react'
+import {FormWrapper, ServiceManualListItems} from 'NewService/components/FormElements'
+import type {FormProps} from 'NewService/types'
+
+const ServiceManualForm = ({formActionPath}: {formActionPath: string}) => {
+  const formProps: FormProps = {
```

Comment:
No need to import the type, they are inferred when importing `FormWrapper`. Also no need to overwork things, there are not many props and the JSX is small, you could set the props directly.